### PR TITLE
Add Xuening to roster_names.md

### PR DESCRIPTION
**Pull request by Xuening99**
#5

### CODE REVIEW

**Comment by mitchphillipson** [comment]
This could use a better title. Like "Add Xuening to roster_names.md"

File: roster_names.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 | Name | 
 | ---  | 
-| Mitch Phillipson | 
\ No newline at end of file
+| Mitch Phillipson | 
+
+| Name |
+| ---  |
```
Comment (mitchphillipson) [inline roster_names.md:6]: You don't need these two lines. They will make a new table. Remove these and put your name under mine

**Comment by mitchphillipson** [comment]
Why do you have two PRs? Try to resolve the conflicts on this one.

**Comment by Xuening99** [comment]
I have changed the code in this one and have the updated ones in commit and the file changed should show my changes. I also closed the second pull request. Hope I can get this right this time. Thank you

**Comment by mitchphillipson** [comment]
image

This is what I see in the merge conflict. Click the `Resolve conflicts` button and delete the `<<< main` and all lines after you name. Then click the `mark as resolved` button.

**Comment by mitchphillipson** [comment]
> I have changed the code in this one and have the updated ones in commit and the file changed should show my changes. I also closed the second pull request. Hope I can get this right this time. Thank you

I'm making this a little more challenging than it is normally. You're doing just fine.

**Comment by Xuening99** [comment]
> > I have changed the code in this one and have the updated ones in commit and the file changed should show my changes. I also closed the second pull request. Hope I can get this right this time. Thank you
> 
> I'm making this a little more challenging than it is normally. You're doing just fine.



> > I have changed the code in this one and have the updated ones in commit and the file changed should show my changes. I also closed the second pull request. Hope I can get this right this time. Thank you
> 
> I'm making this a little more challenging than it is normally. You're doing just fine.



> > I have changed the code in this one and have the updated ones in commit and the file changed should show my changes. I also closed the second pull request. Hope I can get this right this time. Thank you
> 
> I'm making this a little more challenging than it is normally. You're doing just fine.

Thank you! And I also resolved the conflict now on my end.

**Comment by mitchphillipson** [comment]
I still see the conflict, did you commit after resolving the conflict?

**Comment by Xuening99** [comment]
> I still see the conflict, did you commit after resolving the conflict?

Sorry, I didn't commit the first time. Does it work now?

**Comment by mitchphillipson** [comment]
Not only did it work, I'm going to merge and resolve your issue. You're done!

**Comment by Xuening99** [comment]
> Not only did it work, I'm going to merge and resolve your issue. You're done!

Thank you!

**Pull request merged**